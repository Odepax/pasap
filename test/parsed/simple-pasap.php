<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8">
		<title>Pasap Test</title>
		<css at="/asset/Jean-Joseph-Montaigne.css"/>
		<less js="/asset/less.js" files="/asset/one.less, /asset/two.less, /asset/three.less"/>
	</head>
	<body>
		<h1>Wow</h1>
		<p>
			Lorem ipsum dolor sit amet, consectetur adipisicing elit.
			Aut consequuntur cupiditate dicta exercitationem facilis nulla
			obcaecati repellendus, repudiandae sunt tempora voluptate voluptates?
			Accusamus at dignissimos labore nesciunt nulla, sit sunt?
		</p>
	</body>
</html>
