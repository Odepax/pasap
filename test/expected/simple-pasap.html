<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8"/>
		<title>Pasap Test</title>
		<link rel="stylesheet" href="/asset/Jean-Joseph-Montaigne.css"/>
		<link rel="stylesheet/less" href="/asset/one.less"/>
		<link rel="stylesheet/less" href="/asset/two.less"/>
		<link rel="stylesheet/less" href="/asset/three.less"/>

		<script src="/asset/less.js"></script>
	</head>
	<body>
		<h1>Wow</h1>
		<p>
			Lorem ipsum dolor sit amet, consectetur adipisicing elit.
			Aut consequuntur cupiditate dicta exercitationem facilis nulla
			obcaecati repellendus, repudiandae sunt tempora voluptate voluptates?
			Accusamus at dignissimos labore nesciunt nulla, sit sunt?
		</p>
	</body>
</html>
